<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>581</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/pics1/img/front.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>551</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/pics2/img/newbg.jpeg</pixmap>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>292</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="blue">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/pics1/img/blue.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="blue_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>256</y>
      <width>67</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/pics1/img/blue.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="blue_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/pics1/img/blue.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="red">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/pics1/img/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="red_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>67</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/pics1/img/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="red_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>266</y>
      <width>67</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/pics1/img/red.png</pixmap>
    </property>
   </widget>
   <widget class="QLCDNumber" name="record">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SCORE</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_2</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>red</zorder>
   <zorder>red_2</zorder>
   <zorder>red_3</zorder>
   <zorder>blue</zorder>
   <zorder>blue_2</zorder>
   <zorder>blue_3</zorder>
   <zorder>record</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
